src/lib.rs

```diff
@@ -1610,6 +1615,20 @@ impl RtcConfig {
         self
     }
 
+    /// Set DTLS certification.
+    pub fn set_dtls_certification(mut self, dtls_cert: DtlsCert) -> Self {
+        self.dtls_cert = dtls_cert;
+        self
+    }
+
+    /// Toggle certificate fingerprint verification.
+    ///
+    /// By default the certificate fingerprint is verified.
+    pub fn set_certificate_fingerprint_verification(mut self, enabled: bool) -> Self {
+        self.certificate_fingerprint_verification = enabled;
+        self
+    }
```

Comment:
1. For consistent naming, rename this to `set_fingerprint_verification` (in all places).
2. Make a getter `fingerprint_verification` that has a doc test which checks the default is `true`.

Reply:
Done